modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFClaimForm/components/MDFClaimPage/schema/yup.ts

```diff
@@ -152,15 +127,12 @@ const claimSchema = object({
 							)
 							.test(
 								'fileType',
-								validateDocument.listOfLeadsDocuments.message,
-								(listOfQualifiedLeads) => {
-									if (
-										listOfQualifiedLeads &&
-										!listOfQualifiedLeads.id
-									) {
-										return listOfQualifiedLeads
-											? validateDocument.listOfLeadsDocuments.types.includes(
-													listOfQualifiedLeads.type
+								validateDocument.document.message,
+								(eventProgram) => {
+									if (eventProgram && !eventProgram.id) {
+										return eventProgram
+											? validateDocument.document.types.includes(
+													eventProgram.type
```

Comment:
Improve this logic, the ternary it's not necessary since the same condition is validate in if statement